themes/default/content/blog/announcing-6-0-of-the-pulumi-aws-classic-provider/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Announcing 6.0 of the Pulumi AWS Classic Provider"
+date: 2023-08-29T09:00:00-04:00
+draft: false
+meta_desc: "Announcing 6.0 of the Pulumi AWS Classic Provider, including SDK size optimization, TF plugin framework support, new resources, and more."
+meta_image: meta.png
+authors:
+    - mikhail-shilkov
+    - monica-rodriguez
+tags:
+    - aws
+    - providers
+
+---
+
+We are excited to announce 6.0 of the [Pulumi AWS Classic provider](https://www.pulumi.com/registry/packages/aws/). The AWS Classic provider is the most heavily used provider across the entire Pulumi ecosystem, and offers access to the full surface area of the upstream Terraform AWS Provider from within Pulumi projects in all supported Pulumi languages. The [6.0 release](https://github.com/pulumi/pulumi-aws/releases/tag/v6.0.2) brings a substantial set of fixes and improvements to the provider, including a number of breaking changes as part of the major version release.
+
+This blog post walks you through the list of notable changes in the new major version.
+
+<!--more-->
+
+{{% notes type="info" %}}
+Join Pulumi on Thursday, September 7, 2023 for a live workshop on [getting started with infrastructure as code on AWS](/resources/getting-started-with-iac-pulumi-aws/), which will feature the new AWS 6.0 provider.
+{{% /notes %}}
+
+### Terraform Plugin Framework Support
+
+More and more Terraform resources are migrating to the [Terraform Plugin Framework](https://developer.hashicorp.com/terraform/plugin/framework). We are excited to announce that 6.0 supports all resources implemented with both Plugin SDK and Plugin Framework technologies. This means that users get access to the entire API of the upstream provider.
+
+### New Resources and Functions
+
+![aws_api](aws_api.png)
+
+Since the previous major release of 5.0, we have shipped 57 additional minor and patch releases, enabling you to keep up with the latest and greatest enhancements. The new major version of the AWS Classic provider brings 56 new resources and 23 new functions. Additions include resources for Audit Manager, Chime, Fin Space, Global Accelerator, Open Search, Quicksight, Resource Explorer, Route53, Transfer, and many other AWS services.
```

Comment:
```suggestion
Since the previous major release of v5.0, we have shipped 57 additional minor and patch releases, enabling you to keep up with the latest and greatest enhancements. The new major version of the AWS Classic provider brings 56 new resources and 23 new functions. Additions include resources for Audit Manager, Chime, Fin Space, Global Accelerator, Open Search, Quicksight, Resource Explorer, Route53, Transfer, and many other AWS services.
```

Reply:
We're standardizing on 5.0, 6.0, etc.